view/adminhtml/web/js/rename.js

```diff
@@ -11,25 +11,40 @@ define([
             tree          = treeContainer.jstree(true);
 
         treeContainer.on("changed.jstree", function(e, data) {
-            if (options.type === 'node_classes') {
-                var node = data.instance.get_node(data.selected);
+            var node = data.instance.get_node(data.selected);
```

Comment:
Are you sure it's safe to keep it outside the condition?

Reply:
yes it works fine, and no need to repeat declaration for each field type